paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -108,7 +108,13 @@ class PaymentSheet internal constructor(
          *
          * If set, PaymentSheet will pre-populate the form fields with the values provided.
          */
-        var defaultBillingDetails: BillingDetails? = null
+        var defaultBillingDetails: BillingDetails? = null,
+
+        /**
+         * Whether or not to allow payment methods that have a delay before payment or setup
+         * completes, like most bank debits (e.g SEPA) and vouchers (e.g. OXXO, Konbini, Boleto).
```

Comment:
Do we have support for kobini and Boleto on Android? Also should we include OXXO in `PaymentMethod#hasDelayedSettleMent`?

Probably something unrelated, but when confirming an OXXO the PI's status might be `RequiresAction` instead of `Processing`. (therefore not fall into the logic you changed here) - if that's the case, it seems the boolean introduced here won't impact OXXO at all, am i missing anything here?